packages/communication-react/CHANGELOG.stable.md

```diff
@@ -4,103 +4,44 @@ This log was last generated on Thu, 07 Sep 2023 02:42:30 GMT and should not be m
 
 <!-- Start content -->
 
-## [1.8.0](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.8.0)
+## [1.8.0](https://github.com/azure/communication-ui-library/tree/1.8.0)
 
 Thu, 07 Sep 2023 02:42:30 GMT 
-[Compare changes](https://github.com/azure/communication-ui-library/compare/@azure/communication-react_v1.8.0-beta.1...@azure/communication-react_v1.8.0)
+[Compare changes](https://github.com/azure/communication-ui-library/compare/1.8.0-beta.1...1.8.0)
+
+### Capabilities - General Availability
```

Comment:
Did raise hand and background effects also hit GA in this release?

Reply:
Not sure if it is on feature driver or release driver to create these changelog entries...
Maybe release driver asks feature teams for a snippet to paste here

Reply:
I see. I was not sure if we have to snippet for features that are in beta going to stable. But it makes sense to do so. I can just copy the snippet from the beta changelog for each feature being promoted to GA. Does that sound okay?